src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -192,25 +194,38 @@ public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Monu
     }
 
     /**
-     * Create many Monuments based on the specified CSV file contents
-     * @param csvContents - List of Strings, where each String is a CSV row
-     * @return BulkCreateResult - Object containing information about the Bulk Monument Create operation
+     * Validate which rows in the specified .csv (or .csv within .zip) file are valid
+     * @param request - Contains the field mapping and the file to process
+     * @return BulkCreateResult - Object representing the results of the Bulk Monument Create operation
```

Comment:
Minor: Should be changed to something like `Object representing the results of the Bulk Monument Validate operation`